node/tests/TestUtilities.ts

```diff
@@ -78,13 +78,16 @@ export function transactionTest(
     baseTransaction.hdel(key4, [field]);
     baseTransaction.hmget(key4, [field]);
     baseTransaction.hexists(key4, field);
-    baseTransaction.lpush(key5, [field + "1", field + "2", field + "3"]);
```

Comment:
why did you reorder calls here?

Reply:
wanted the python and node tests to look more similar, I will return the original order